src/internal/observable/combineLatest.ts

```diff
@@ -50,7 +50,7 @@ export function combineLatest<R>(...observables: Array<ObservableInput<any> | ((
  * computes a formula using the latest values from all the inputs, then emits
  * the output of that formula.</span>
  *
- * <img src="combineLatest.png">
+ * ![](combineLatest.png)
```

Comment:
Can we add text in the `[]` part of these? It should set the title of the image tag and help with a11y issues.  `![combineLatest marble diagram](combineLatest.png)`

Reply:
@benlesh The alt attributes are generated by the post-processor because I didn't want to copy&paste them everywhere so when there's no `alt` it'll be filled automatically here: 

https://github.com/martinsik/rxjs/blob/4731b753e6de81900e1f029c2a61d699b9ce6881/docs_app/tools/transforms/angular-api-package/post-processors/embedMarbleDiagrams.js#L34-L36